diagnostic.md

```diff
@@ -150,7 +181,13 @@ div {
 ```
 
 <!-- your answer starts here -->
+Div one will be on the right side of the page.
+The the following divs will appear left to right:
+divTwo
+divThree
+divFive
 
+Div four will appear below all the divs on the left. 
```

Comment:
Close. .divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5